pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -385,6 +392,24 @@ func (pe *StandardSecurityPolicyEnforcer) enforceEnvironmentVariablePolicy(conta
 	return nil
 }
 
+func (pe *StandardSecurityPolicyEnforcer) enforceWorkingDirPolicy(containerID string, workingDir string) error {
+	possibleIndices := possibleIndexesForID(containerID, pe.ContainerIndexToContainerIds)
```

Comment:
Should probably stick to either indexes or indices for both

Reply:
changed to `indices`